feature-group-definitions/media-capture.yml

```
@@ -0,0 +1,42 @@
+name: Media capture
+description: The `navigator.mediaDevices.getUserMedia()` API requests access to devices that produce streams of data, such as microphones or video cameras.
```

@ddbeck WDYT about making it more concrete?

```suggestion
description: The `navigator.mediaDevices.getUserMedia()` API requests access to devices that produce audio or video streams, such as microphones or video cameras.
```

---

Yes, that's an good edit. 👍